packages/s3wrapper/requirements/_test.txt

```diff
@@ -2,39 +2,39 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file=_test.txt _test.in
+#    pip-compile --output-file=requirements/_test.txt requirements/_test.in
 #
 astroid==2.4.0            # via pylint
 attrs==19.3.0             # via pytest, pytest-docker
-certifi==2020.4.5.1       # via -r _base.txt, minio, requests
+certifi==2020.4.5.1       # via -r requirements/_base.txt, minio, requests
 chardet==3.0.4            # via requests
-configparser==5.0.0       # via -r _base.txt, minio
-coverage==5.1             # via -r _test.in, coveralls, pytest-cov
-coveralls==2.0.0          # via -r _test.in
+configparser==5.0.0       # via -r requirements/_base.txt, minio
+coverage==5.1             # via -r requirements/_test.in, coveralls, pytest-cov
+coveralls==2.0.0          # via -r requirements/_test.in
 docopt==0.6.2             # via coveralls
 idna==2.9                 # via requests
 importlib-metadata==1.6.0  # via pluggy, pytest
 isort==4.3.21             # via pylint
 lazy-object-proxy==1.4.3  # via astroid
 mccabe==0.6.1             # via pylint
-minio==5.0.10             # via -r _base.txt
+minio==5.0.10             # via -r requirements/_base.txt
 more-itertools==8.2.0     # via pytest
 packaging==20.3           # via pytest
 pluggy==0.13.1            # via pytest
 py==1.8.1                 # via pytest
-pylint==2.5.0             # via -r _test.in
+pylint==2.5.0             # via -r requirements/_test.in
```

Comment:
be careful with that one since it has a bug. maybe better use pylint !=2.5.0 see [https://github.com/PyCQA/pylint/issues/3524](https://github.com/PyCQA/pylint/issues/3524)